WalletWasabi.Tests/UnitTests/RpcTests.cs

```diff
@@ -132,6 +132,19 @@ public void BuildTransactionWithFees()
 		Assert.Throws<InvalidOperationException>(() => BuildTransaction(feeTarget: 1008));
 	}
 
+	[Fact]
+	public void CanParseRequest()
+	{
+		var request = "['{jsonrpc:2.0,id:1,method:getwalletinfo}']";
```

Comment:
I think the problem was that you need to wrap the method key's value inside `" "`.
```suggestion
		var request = "{jsonrpc:2.0,id:1,method:\"getwalletinfo\"}";
```